### PR TITLE
Finished tuning of gulpfile.js

### PR DESCRIPTION
**Pull request by RSRaven**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        css: 'client_src/scss/**/*.scss',
+        img: 'client_src/img/**/*.*',
+        fonts: 'client_src/fonts/**/*.*'
+    },
+    clean: 'client_build'
+};
+
+var serverConfig = {
+    server: {
+        baseDir: "client_build"
```
Comment (puzankov) [inline gulpfile.js]: 'client_build'

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+               .pipe(gulp.dest(path.build.html))
+               .pipe(reload({stream: true}));
+});
+
+gulp.task('build:css', function () {
+    return gulp.src(path.src.css)
+               .pipe(sourcemaps.init())
+               .pipe(sass())
+               .pipe(prefixer())
+               .pipe(gulpif( argv.prod, cssnano() ))
```
Comment (puzankov) [inline gulpfile.js]: remove if

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+               .pipe(sourcemaps.write())
```
Comment (puzankov) [inline gulpfile.js]: add if

File: gulpfile.js [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+               .pipe(gulp.dest(path.build.css))
+               .pipe(reload({stream: true}));
+});
+
+gulp.task('build:js', function () {
+    return gulp.src(path.src.js)
+               .pipe(rigger())
+               .pipe(sourcemaps.init())
+               .pipe(gulpif( argv.prod, uglify()))
```
Comment (puzankov) [inline gulpfile.js]: same as style if

**Comment by RSRaven** [comment]
All notices fixed. Now this branch can be merged.

**Pull request merged**